AtlasUtil/include/AtlasUtil/Annotate.h

```diff
@@ -112,7 +171,7 @@ inline int64_t GetBlockID(llvm::BasicBlock *BB)
 
 inline int64_t GetValueID(llvm::Value *val)
 {
-    int64_t result = -1;
+    int64_t result = -2;
```

Comment:
This was moved to -2 because of a bug I found when tikSwapping bubbleSort. 
 - 0 is a valid ValueID that may be used as a kernel function argument
 - (-1) is the valueID used for the constant first argument of each kernel function
 - (-2) is used as the unset value

Reply:
Now explicitly stated in Annotate.h with IDState enum